server/php-engine.cpp

```diff
@@ -2098,6 +2098,19 @@ int main_args_handler(int i, const char *long_option) {
     case 2018: {
       return read_option_to(long_option, 0.0, 10.0, vk::singleton<LeaseContext>::get().rpc_stop_ready_timeout);
     }
+    case 2019: {
+      // TODO REMOVE ME!
+      OPTION_ADD_DEPRECATION_MESSAGE("--job-workers-num");
+      return parse_numeric_option(long_option, 0, int{WorkersControl::max_workers_count} / 2, [](int job_workers_num) {
+        auto &control = vk::singleton<WorkersControl>::get();
+        const uint16_t total_workers = control.get_total_workers_count();
+        // expect that -f options is set in advance
+        assert(total_workers);
+        control.set_total_workers_count(total_workers + job_workers_num);
```

Comment:
Let's add an assertion that `total_workers < MAX_WORKERS`

Reply:
make sense, added